src/components/companyMatchPage/index.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from "react";
+import { Col, Row, Container, Button, Image, Badge } from "react-bootstrap";
+import male from "../../assets/images/male.png";
+const CompanyMatchPage = () => {
+  return (
+    <div className="Container">
+      <div className="Row mb-2">
+        {/* <div className="Col-md-2"></div> */}
+        <div className="Col-md-12">
+          <a href="#" className="text-dark">
+            <div className="card shadow">
+              <div className="card-body">
+                <div className="row">
+                  <div className="col-xs-1 col-sm-2 col-md-1 mr-3">
+                    <Image src={male} />
+                    <br></br>
+                  </div>
+                  <div className="col-xs-8 col-sm-7 col-md-8">
+                    <h5>Car System Alerts</h5>
+                    <p>Check the battery and alternator conditions</p>
```

Comment:
Use real attributes like name. 
Add the Posts, Chats, and Match links and the View and Chat buttons into the page.